std_msgs/Header header

float64 latitude   
float64 longitude
float64 height    
float32 undulation

float64 north_velocity  
float64 east_velocity  
float64 up_velocity   
float64 roll            
float64 pitch           
float64 azimuth 

float32 latitude_std
float32 longitude_std
float32 height_std
float32 north_velocity_std
float32 east_velocity_std
float32 up_velocity_std

float32 roll_std
float32 pitch_std
float32 azimuth_std
